code/modules/mob/new_player/sprite_accessories/ipc/ipc_face.dm

```diff
@@ -1,8 +1,9 @@
 /datum/sprite_accessory/hair/ipc
 	icon = 'icons/mob/sprite_accessories/ipc/ipc_face.dmi'
 	species_allowed = list("Machine")
-	glasses_over = 1
-	models_allowed = list("Bishop Cybernetics mtr.", "Hesphiastos Industries mtr.", "Morpheus Cyberkinetics", "Ward-Takahashi mtr.", "Xion Manufacturing Group mtr.", "Shellguard Munitions Monitor Series")
+	glasses_over = TRUE
+	//Any and all IPC Monitors with a screen
+	models_allowed = list("Bishop Castle", "Industrial Revolution", "Morpheus Cyberkinetics", "Ward-Takahashi Classic", "Xion Original", "Shellguard Munitions Standard Series")
```

Comment:
The names. I know. I know. Okay, I had fun with the names. Seriously, same thing with the "Amputated". I grew tired of opening the prostheses menu for IPCs and seeing something like "Bishop Cyberkinetics mtr.". At first, it was just getting rid of the abbreviations. Then it was .... then it was something _more_. Something fun, something to bring flavor to the game.

These monitors are the ones that can use screens. I double checked the dmi files.